roles/estacao/tasks/060-configura-browsers.yml

```
@@ -44,8 +44,8 @@
     # Referencia para policies do Firefox:
     # https://mozilla.github.io/policy-templates/
     # # https://mozilla.github.io/policy-templates/#managedbookmarks
-    - src: "policies-firefox.j2"
-      dest: "/usr/lib/firefox/distribution/policies.json"
+    - src: "policies.json"
+      dest: "/etc/firefox/policies"
```

O template de origem continua o mesmo, muda só o destino, né?

```suggestion
    - src: "policies-firefox.j2"
      dest: "/etc/firefox/policies/policies.json"
```